runtime/autoload/health/provider.vim

```diff
@@ -243,7 +243,9 @@ function! s:check_python(version) abort
   let python_multiple = []
 
   if exists(loaded_var) && !exists('*provider#'.pyname.'#Call')
-    call health#report_info('Disabled. '.loaded_var.'='.eval(loaded_var))
+    call health#report_info('Disabled due to: `'.loaded_var.'='.eval(loaded_var).'`')
```

Comment:
Just a small change should be enough to clear up confusion:

    call health#report_info('Disabled: '.loaded_var.' is '.eval(loaded_var))